src/context.cpp

```diff
@@ -857,7 +881,7 @@ namespace librealsense
 
         platform::backend_device_group get_device_data() const override
         {
-            return platform::backend_device_group{};
+            return platform::backend_device_group{ { platform::playback_device_info{ _dev->device_info().topic_root } } };
```

Comment:
What does "playback_device_info" means here?

Reply:
`platform::backend_device_group` have a constructor that gets a `platform::playback_device_info` for software devices.
For actual playback devices it is the file path.
For the DDS device I use the topic root